aws/table_aws_route53_record.go

```diff
@@ -20,10 +20,15 @@ func tableAwsRoute53Record(_ context.Context) *plugin.Table {
 		Name:        "aws_route53_record",
 		Description: "AWS Route53 Record",
 		List: &plugin.ListConfig{
+			IgnoreConfig: &plugin.IgnoreConfig{
```

Comment:
@bigdatasourav Do we still need this error handling as well? What case does this handle?